distribution/tools/keystore-cli/src/test/java/org/elasticsearch/common/settings/AddStringKeyStoreCommandTests.java

```diff
@@ -176,16 +176,6 @@ public void testMissingSettingName() throws Exception {
         assertThat(e.getMessage(), containsString("The setting name can not be null"));
     }
 
-    public void testUpperCaseInName() throws Exception {
```

Comment:
We shoudl keep this test, but rename it and use a special character instead of uppercase.